Source/FirebaseNotificationsAddon.swift

```diff
@@ -137,7 +160,19 @@ open class FirebaseNotificationsAddon: NSObject, HaloNotificationsAddon, HaloLif
     open func application(_ app: UIApplication, didReceiveRemoteNotification userInfo: [AnyHashable: Any], core: CoreManager, userInteraction user: Bool, fetchCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
 
         let notification = HaloNotification(userInfo: userInfo)
-        
+    
+        let haloNotificationEvent : HaloNotificationEvent = HaloNotificationEvent(device: (core.device?.alias)!, schedule: userInfo["scheduleId"] as! String, action: EventType.receipt.rawValue)
```

Comment:
Change this to if let instead of using force unwraps just to be on the safe side of things